sanic/http.py

```diff
@@ -380,7 +374,7 @@ def head_response_ignored(self, data: bytes, end_stream: bool) -> None:
         HEAD response: body data silently ignored.
         """
         if end_stream:
-            self.response_func = None
+            self.q = None
```

Comment:
What is this?

Reply:
Looks like a typo or change that wasnt supposed to be committed